CHANGELOG.rst

```diff
@@ -6,6 +6,11 @@ dcicwrangling
 Change Log
 ----------
 
+2.0.1
+=====
+
+* Update ubuntu version for running github workflows
```

Comment:
Should maybe say to what version, though that's minor. More an observation on how people use changelogs. If someone were searching later for some particular thing, it would most likely be "when did we start using this version", so writing changelogs that answer that specific question is usually most important just as a style. Will people care that much about this? Probably not.